pmoves/services/flute-gateway/providers/ultimate_tts.py

```
@@ -0,0 +1,401 @@
+                timeout=60.0,
+            )
+            # Check for success in SSE response
+            for line in result_resp.iter_lines():
+                if line.startswith("data:"):
+                    data = json.loads(line[5:])
+                    if isinstance(data, list) and len(data) > 0:
+                        status = str(data[0]) if data[0] else ""
+                        if "✅" in status or "Loaded" in status:
+                            logger.info("Ultimate-TTS %s model loaded", engine)
+                            return True
+            return True  # Assume loaded if no error
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)